tests/data/ekinops.json

```diff
@@ -0,0 +1,10572 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.20044",
+                    "sysDescr": "Ekinops 360 Platform, C600HC, Release 6.3.900",
+                    "sysContact": null,
+                    "version": " Release 6.3.900",
```

Comment:
Maybe want to use trim() to get rid of the spaces

Reply:
fixed